README.md

```diff
@@ -8,7 +8,7 @@ Caffe, Rust and numerous research papers and brings modularity, performance and
 portability to deep learning. Leaf is lean and tries to introduce minimal
 technical debt to your stack.
 
-Leaf is a few months old, but thanks to its architecture and Rust already one of
+Leaf is a few months old, but thanks to its architecture and Rust it's already one of
```

Comment:
Is that change really necessary for understandability? I prefer the old way because it sounds more crispy, but I agree to to change if it's not understandable or blatantly wrong.